sibilant.js

```diff
@@ -80,8 +101,7 @@ var audioContext = null;
 class Sibilant {
   constructor (stream, options) {
     options = options || {};
-    var self = this;
-    // var useBandPass = (options.useBandPass || true)
+    const self = this;
```

Comment:
Actually `self` is never used (anywhere in this file) so it could be deleted altogether.